src/Symfony/Component/Routing/Generator/CompiledUrlGenerator.php

```diff
@@ -21,18 +21,33 @@
 class CompiledUrlGenerator extends UrlGenerator
 {
     private $compiledRoutes = [];
+    private $compiledAliases = [];
     private $defaultLocale;
 
     public function __construct(array $compiledRoutes, RequestContext $context, LoggerInterface $logger = null, string $defaultLocale = null)
     {
-        $this->compiledRoutes = $compiledRoutes;
+        // TODO: To be removed in Symfony 6.0
+        if (!isset($compiledRoutes['routes'])) {
```

Comment:
IMHO the test is wrong, what if I have a route name `routes` with the old format.
You should probably assert that the array contains `routes`, `aliases` and nothing else

Reply:
Adressed.